lib/headquarters/client.rb

```diff
@@ -0,0 +1,13 @@
+require 'headquarters/api'
+
+module Headquarters
+  class Client < API
+    namespace :members
```

Comment:
Let's go with it then. But we have to acknowledge that this is creating a DSL and a convention for naming the classes.